Source/Libraries/NetCore/NetCoreMessages.cs

```diff
@@ -7,7 +7,7 @@ namespace RTCV.NetCore
     [MemberConfig(TargetMember.All)]
     public abstract class NetCoreMessage
     {
-        public string Type;
+        public string Type { get; set; }
```

Comment:
We need to evaluate any potential differences with this being a property over a field when it comes to serialization. I don't _forsee_ any issues, but we might want to split this one off and do proper testing to make sure it doesn't introduce regressions.

Reply:
Done